src/config/dark-sites.config

```diff
@@ -726,6 +726,7 @@ polsatboxgo.pl
 polsatgo.pl
 pony.tube
 pool.pm
+popper.js.org
```

Comment:
I don't find the start page that dark, quite flashy when you open the site.

Reply:
This is weird, can you provide some efficient ways to check? incognito mode is not enough I guess.